packages/design-system/src/components/keyboard/index.tsx

```diff
@@ -291,9 +262,9 @@ function resolveKeySpec(keyDict, keyNameOrSpec) {
     dialog = false,
     allowDefault = false,
   } = keySpec;
-  const mappedKeys = []
+  const mappedKeys = ([] as string[])
```

Comment:
Not a big fan of this because it's ugly. Maybe `new Array<string>()` is cleaner?